conftest.py

```diff
@@ -90,8 +90,8 @@ def fake():
         'mark': 'enable_search',
         'replacement': mock.MagicMock()
     },
-    'website.search.elastic_search': {
-        'mark': 'enable_search',
```

Comment:
yeah I'm surprised tests still pass with this removed, I'd want to keep it deleted as that is the case, don't want to mock stuff I don't need to, especially since I had to remove it to have things work locally.

Reply:
IIRC, this was intended not to make tests pass so much as to speed the CI build up -- not sending numerous updates to ES if the unit test didn't care about interacting with search.

I'm a little surprised it didn't seem to affect CI runtime, but a potential explanation might be that the Search Improvements project largely mitigated this.